karapace/backup/cli.py

```diff
@@ -40,6 +40,11 @@ def parse_args() -> argparse.Namespace:
         p.add_argument("--poll-timeout", help=PollTimeout.__doc__, type=PollTimeout, default=PollTimeout.default())
 
     parser_get.add_argument("--use-format-v3", action="store_true", help="Use experimental V3 backup format.")
+    parser_get.add_argument(
+        "--replication-factor",
+        help="Use a specified replication factor instead of retrieving it from the current topic state",
+        type=int,
+    )
```

Comment:
issue: We discussed making this a required argument with a special `--replication-factor=auto` mode, because the behavior when not providing a value isn't reliable. It still seems like a good idea to do that, and to document in the help-text that it must be used with caution.

Reply:
Resolving this because we agreed to have the replication factor mandatory for v3 format.